code/datums/gamemodes/crash.dm

```diff
@@ -199,3 +199,19 @@
 		return //Things are balanced, no burrowed needed
 	xeno_job.add_job_positions(1)
 	xeno_hive.update_tier_limits()
+
+/datum/game_mode/infestation/crash/get_total_joblarvaworth(list/z_levels, count_flags)
+	. = 0
+
+	for(var/i in GLOB.human_mob_list)
+		var/mob/living/carbon/human/H = i
+		if(!H.job)
+			continue
+		if(count_flags & COUNT_IGNORE_HUMAN_SSD && (H.stat != DEAD && !H.client))
+			continue
```

Comment:
Then dead marines do not count ...

Reply:
I mainly want to avoid that when some one dies and leaves, they would not have a client in there body there for count. So this should only happen if they ARENT dead AND dont have a client. would mean they are SSD. if they are dead then it will leave this guard clause.